go-controller/pkg/ovn/logical_switch_manager/logical_switch_manager.go

```diff
@@ -31,6 +31,10 @@ func NewL2SwitchManager() *LogicalSwitchManager {
 	}
 }
 
+func NewL2SwitchManagerForUserDefinedPrimaryNetwork() *LogicalSwitchManager {
```

Comment:
nit: func comment on why this is different from  NewL2SwitchManager and why we needed this wrapper

Reply:
Done.